ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -406,34 +667,91 @@ class ServicesHomeScreen extends React.Component<Props, State> {
 }
 
 const mapStateToProps = (state: GlobalState) => {
-  const { services } = state.entities;
-
   const potUserMetadata = userMetadataSelector(state);
   // TODO: disable selection of areas of interest if the user metadata are not loaded
   // (it causes the new selection is not loaded) https://www.pivotaltracker.com/story/show/168312476
   const userMetadata = pot.getOrElse(potUserMetadata, undefined);
 
-  const isAnyServiceLoading =
-    Object.keys(services.byId).find(k => {
-      const oneService = services.byId[k];
-      return oneService !== undefined && pot.isLoading(oneService);
-    }) !== undefined;
+  const localServicesSections = localServicesSectionsSelector(state);
+  const selectableOrganizations = localServicesSections.map(
+    (section: ServicesSectionState) => {
+      return {
+        name: section.organizationName,
+        fiscalCode: section.organizationFiscalCode
+      };
+    }
+  );
 
-  const isLoading =
-    pot.isLoading(state.entities.services.visible) ||
-    isAnyServiceLoading ||
-    pot.isLoading(potUserMetadata);
+  const localSections = selectedLocalServicesSectionsSelector(state);
+  const nationalSections = nationalServicesSectionsSelector(state);
+  const allServicesSections = notSelectedServicesSectionsSelector(state);
+
+  const getTabSevicesId = (
+    tabServices: ReadonlyArray<ServicesSectionState>
+  ) => {
+    // tslint:disable-next-line: readonly-array
+    const tabServicesId: string[] = [];
+    tabServices.forEach(section => {
+      section.data.forEach(service => {
+        if (pot.isSome(service)) {
+          tabServicesId.push(service.value.service_id);
+        }
+      });
+    });
+    return tabServicesId;
+  };
+
+  // tslint:disable-next-line: readonly-array
+  const tabsServicesId: { [k: number]: string[] } = {
```

Comment:
```suggestion
  const tabsServicesId: { [k: number]: ReadonlyArray<string> } = {
```
`tslint:disable-next-line` can be removed

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1269/commits/383772845fa1b1dd59a4d61d5512ba2520b8f0ce